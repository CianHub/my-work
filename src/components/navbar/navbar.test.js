// navbar.navbar.test.js
import React from "react";
import renderer from "react-test-renderer";
import Navbar from "./navbar";

test("Navbar is rendered", () => {
  const component = renderer.create(<Navbar />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
