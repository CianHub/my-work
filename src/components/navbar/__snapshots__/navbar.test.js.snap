// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar is rendered 1`] = `
<div>
  <nav
    className="navbar navbar-expand-lg navbar-light "
    id="mainNav"
  >
    <div
      className="container mx-auto "
    >
      <a
        className="navbar-brand"
        href="#jumbotron"
      >
        Cian's Work
      </a>
      <button
        aria-controls="navbarResponsive"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler navbar-toggler-right"
        data-target="#navbarResponsive"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse "
        id="navbarResponsive"
      >
        <ul
          className="navbar-nav ml-auto"
        >
          <li
            className="nav-item"
          >
            <a
              className="nav-link js-scroll-trigger"
              href="#about"
            >
              About
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link js-scroll-trigger"
              href="#projects"
            >
              Project
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link js-scroll-trigger"
              href="#contact"
            >
              Contact
            </a>
          </li>
          <li
            className="nav-item"
          >
            <a
              className="nav-link js-scroll-trigger"
              href="https://drive.google.com/file/d/1XjSEkJNjIpcjNdtNM3osbvoj3HVqUCIb/view?usp=sharing"
              target="_blank"
            >
              Download CV
            </a>
          </li>
        </ul>
      </div>
    </div>
  </nav>
</div>
`;
